NetKAN/CryoEngines-LFO.netkan

```diff
@@ -1,16 +1,5 @@
 {
     "spec_version": "v1.4",
     "identifier": "CryoEngines-LFO",
-    "$kref": "#/ckan/spacedock/709",
-    "$vref": "#/ckan/ksp-avc/GameData/CryoEngines/Versioning/CryoEngines.version",
-    "name": "Cryogenic Engines Extras: LFO Patch",
-    "abstract": "Optional patch to make the engines burn KSP's normal liquid fuel and oxidizer mixture instead of the custom liquid hydrogen and oxidizer mixture that they use by default.",
-    "license": "CC-BY-NC-SA-4.0",
-    "x_netkan_epoch": 1,
```

Comment:
This `x_netkan_epoch` property was lost in the transition; it needs to be preserved, or the bot will treat newer versions as old versions.

Reply:
So it's an override on version numbers? eg 
`1.0.` with `x_netkan_epoch": 1` would count as newer than say `14.0` with `x_netkan_epoch": 0`

Reply:
Yes. In this case it was added in #1911 because the 1.81 release was followed by 1.9, and 9 < 81.

Reply:
Mm ok. This is exactly why I couldn't support CKAN before validating and managing all this meta myself. Presumably there would have been a typo in a package somewhere having version as `1.81` versus `1.8.1` but to my knowledge this mod has never exceeded version 0.6.6 so... 🤷‍♂️ 

I restored the field in the host repo.

Reply:
Sorry, typo on my part; the relevant versions were 0.1.81 and 0.1.9. I got confused by where one number stopped and the next began.

![image](https://user-images.githubusercontent.com/1559108/56620308-175b6c00-65ee-11e9-98d2-7741beb0487e.png)

Reply:
Ah yeah, same issue I guess - likely a borked upload to I guess, KerbalStuff back then.